scripts/data/tokenHolders.js

```diff
@@ -0,0 +1,170 @@
+const TOKEN_HOLDERS = [
```

Comment:
We are using this today, but in the future we could fetch that data from a CSV (i[ssue](https://github.com/TECommons/tec-template/issues/2)).